.github/workflows/webserver-release.yml

```diff
@@ -0,0 +1,178 @@
+name: webserver release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to release'
+        required: true
+
+jobs:
+  build-scies:
+    strategy:
+      fail-fast: false
+      matrix:
+        # ubuntu-latest: intel
+        # linux-aarch64: aarch64 (self-hosted)
+        # macos-12: intel
+        # macos-13: apple silicon
+        os: [ubuntu-latest, linux-aarch64, macos-13-xlarge, macos-12-large]
```

Comment:
It's a minor thing, but it bothers me that the order of macos-12 and macos-13 is different from the comment.

Reply:
This is taken from ci.yml.
We will modify ci.yml in the future as well.